fineract-doc/src/docs/en/chapters/architecture/api-performance.adoc

```diff
@@ -0,0 +1,115 @@
+= Introducing Performance Enhancing Read/Write/Batch API Operation Separation
+
```

Comment:
Also, this new file is never referenced in the index file so it doesn't appear anywhere.